### PR TITLE
Remove libmagic

### PR DESCRIPTION
**Pull request by ljyanesm**
Removes libmagic keeping the checks of the magic bits on the files that were provided by this library, this commit removes a dependency both on python and the OS given that python uses the underlying library which is not available in some container images.

### CODE REVIEW

File: Mikado/utilities/file_type.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 max_len = max(len(x) for x in magic_dict)
 
 
 def filetype(filename):
-    with open(filename) as f:
+    with open(filename, "rb") as f:
         file_start = f.read(max_len)
     for magic, ftype in magic_dict.items():
-        if file_start.startswith(magic):
+        if magic == file_start[:len(magic)]:
```
Comment (ljyanesm) [inline Mikado/utilities/file_type.py]: Any reason why this is preferred  to file_start.startswith(magic), startswith conveys the intention in a much clearer way.

Comment (lucventurini) [reply Mikado/utilities/file_type.py]: Startswith does not function with bytes, so it's not an option :-)

Comment (ljyanesm) [reply Mikado/utilities/file_type.py]: https://docs.python.org/3/library/stdtypes.html#bytes.startswith

Comment (lucventurini) [reply Mikado/utilities/file_type.py]: You are right, I must have typoed it when I tested! Amending and merging.

**Comment by lucventurini** [comment]
Before merging I will remove `python-magic` from the requirements etc.

**Pull request merged**